app/core/RPCMethods/RPCMethodMiddleware.ts

```diff
@@ -392,16 +392,17 @@ export const getRpcMethodMiddleware = ({
           networkType && getAllNetworks().includes(networkType);
         let chainId;
 
-        if (isInitialNetwork) {
-          chainId = NetworksChainId[networkType];
+        if (isInitialNetwork && networkType !== RPC) {
```

Comment:
See previous comment
```suggestion
        if (isInitialNetwork) {
```